### DISCUSSION TITLE
Feature: Lambda Powertools

### DISCUSSION THREAD

**Discussion by babyhuey** [logandk/serverless-wsgi#212] [Ideas]
I recently learned about [Lambda Powertools](https://awslabs.github.io/aws-lambda-powertools-python/latest/core/logger/)

It looks like a really great way to add more logging. Would you be interested in accepting a PR that would incorporate adding this if it is installed? I would do a try/except to see if it's installed, and if not, continue, that way people aren't forced to use it if they don't want to

**Comment by logandk** [comment]
Hi @babyhuey, thanks for the link. Looks promising - I would definitely accept the PR you're describing. It sounds like a valuable addition.
